packages/blade/src/components/Tabs/__tests__/Tabs.ssr.test.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Tabs, TabList, TabItem, TabPanel } from '../';
+import renderWithSSR from '~utils/testing/renderWithSSR.web';
+import { Text } from '~components/Typography';
+
+// Mock requestAnimationFrame
+global.requestAnimationFrame = (callback) => {
+  return setTimeout(callback, 0);
+};
+
+describe('<Tabs />', () => {
+  it.skip('should render tabs ssr', () => {
+    const { container, queryAllByRole } = renderWithSSR(
+      <Tabs defaultValue="refunds">
+        <TabList>
+          <TabItem value="payments">Payments</TabItem>
+          <TabItem value="refunds">Refunds</TabItem>
+        </TabList>
+
+        <TabPanel value="payments">
+          <Text>Payments</Text>
+        </TabPanel>
+        <TabPanel value="refunds">
+          <Text>Refunds</Text>
+        </TabPanel>
+      </Tabs>,
+    );
+    expect(queryAllByRole('tab').length).toBe(2);
```

Comment:
can you do an assertion that all the panels are rendered? because for SEO that is what matters the most

Reply:
By default all tabs are rendered which is already being test on every test suit in the normal tests. In here the snapshots will be enough.